README.md

```diff
@@ -0,0 +1,194 @@
+# [Cozy][cozy] cozy-konnector-google-contacts
+
+## What's Cozy?
+
+![Cozy Logo](https://cdn.rawgit.com/cozy/cozy-guidelines/master/templates/cozy_logo_small.svg)
+
+[Cozy] is a platform that brings all your web services in the same private space. With it, your webapps and your devices can share data easily, providing you with a new experience. You can install Cozy on your own hardware where no one's tracking you.
+
+## What's this new konnector?
+
+This konnector let you import or synchronize your google contacts with your cozy.
+
+___Note: here is the TODO list for this konnector to be complete:___
+
+- [ ] use the Google people API
+- [ ] get a Google OAuth2 token
+- [ ] remove the Google OAuth2 form sight (and use cozy-collect instead)
+- [ ] use cozy-collect to setup a Google OAuth2 token
+- [ ] save Google contacts into Cozy
+- [ ] indicate the source of the contacts (for synchronization purpose with multiple Google accounts)
+
+If something is missing, we'll add it later.
+
+### Open a Pull-Request
+
+If you want to work on this konnector and submit code modifications, feel free to open pull-requests! See the [contributing guide][contribute] for more information about how to properly open pull-requests.
+
+### Cozy-konnector-libs
+
+This connector uses [cozy-konnector-libs](https://github.com/cozy/cozy-konnector-libs). You can
+find more documentation about it there.
+
+## Google OAuth2
+
+As we need to generate a Google OAuth2 token, we need a `CLIENT_ID` and `CLIENT_SECRET`. You can find these pieces of information by going to the [Google Developer Console](https://console.developer.google.com/), clicking your project --> APIs & auth --> credentials. For more information about OAuth2, [see here](https://github.com/google/google-api-nodejs-client/#oauth2-client).
+Once acquired, fill the [keys.json](./keys.json) file.
+
+### Test the connector without an accessible cozy-stack
```

Comment:
We must change the word _"test"_ because it ain't no test but working in a development's environment.

Reply:
I agree but this should be a PR in cozy-konnector-template